Actioned changes in BP5 / section 12.2; also tweaked wording on Briti…

PeterParslow opened the pull request:
…sh National Grid (Note in section 8 & section 13.2)

BP5:
- clarify relationship to general data quality; this BP concentrates on positional accuracy, as the only (?) bit that's special to spatial. Noted that the INSPIRE specs don't specify positional accuracy.
- included Kerry's suggested words in 'why' (plus some other examples)

- fix definition of 'absolute positional accuracy' (as per Jeremy's input)
- deleted temporal resolution example (because I don't know how to express it a machine readable way)

Section 8 change - see pull request 666 for discussion: corrected OSGB 1936 to being described as a datum; effectively the same discussion applies in section 13.2 - removed reference to OSGB 1936.

PeterParslow committed:
Actioned changes in BP5 / section 12.2; also tweaked wording on British National Grid (Note in section 8 & section 13.2)

BP5:
- clarify relationship to general data quality; this BP concentrates on positional accuracy, as the only (?) bit that's special to spatial. Noted that the INSPIRE specs don't specify positional accuracy.
- included Kerry's suggested words in 'why' (plus some other examples)

- fix definition of 'absolute positional accuracy' (as per Jeremy's input)
- deleted temporal resolution example (because I don't know how to express it a machine readable way)

Section 8 change - see pull request 666 for discussion: corrected OSGB 1936 to being described as a datum; effectively the same discussion applies in section 13.2 - removed reference to OSGB 1936.

lvdbrink commented:
@PeterParslow I can't determine what changes you made because the diff is not working and there are merge conflicts. Can you either try again, based on the latest index.html, or give me the changes to add in?

@6a6d74 any other solutions?

PeterParslow commented:
I was just going to ask about this, given that 10 days have passed since Jeremy noticed the problem. Speaking as an IT old time, my guess is that something in the download/edit/upload loop has replaced all the carriage returns with carriage return/line feed pairs (or vice versa).
Just re-uploading it didn't help - it just created a separate pull request.

Looking more closely, it isn't that: saving the Raw view (https://raw.githubusercontent.com/w3c/sdw/gh-pages/index.html, with Firefox) & comparing with the local copy created by GitHub for Windows, they both have x0Dx0A for each line break.

GitHub for Windows can show the tracked changes, in my local repository. I'll create another pull request from their, to see if that helps. Well, actually GitHub for Windows wouldn't let me create a pull request, but it did let me commit my change to patch-1 (locally), and re-sync - but the sync failed "would overwrite your uncommitted changes. Please discard or commit your changes". The it had me worried, as it said I had no local files - before finding them again.

Has noone else on the team ever had whatever problem this is? Seems like I over estimated my competence with GitHub, having only ever used the online version on reasonable sized files before (TC211 XML projects).

Linda, I'll take you up on your offer, and email you my changed file (actually, I'll drop it in this comment, after zipping
[index.zip](https://github.com/w3c/sdw/files/946432/index.zip)
) 
- as I say, GitHub for Windows can spot the differences, so perhaps you'll have more luck uploading it somehow!

lvdbrink commented:
There have been other changes since this pull request so I compared the file from your zip with **your** previous version (I think). 

The changes I found are listed below, could you check if this is correct? I have made these changes again in the current draft and will commit them soon. 

_In the intro on data quality (which has been merged with the intro on metadata):_ 
This section is based on the Data Quality section from [[DWBP]] and adds a  best practice specific for spatial data, which concentrates on the accuracy of the positions in the data - how close are they to the actual positions of the real world things?

_Appended to the intro on metadata:_ 
        <p>However, that specification makes no statement about the positional accuracy of the data, 
          so on its own, it is only a useful quality statement for users to whom positional accuracy is not that important.</p>

_In the Why section of former BP5:_ 
Some spatial data applications, such as aircraft navigation, require highly accurate data. For others, such as human navigation, a horizontal accuracy of a few metres is good enough. For yet others, such as overlaying weather forecasts on a map, the map is only giving a general indication of place. If the positional accuracy is published together with the data, the user can determine whether it is appropriate to use for their application. Potentially, this makes existing data more re-usable.

_Removed from former BP5 example list:_ 
                <li>second quarter of the 9th century is <em>approx.</em> 825-850 but could be
                  823-852</li>

_In former BP5 example list:_ 
1 000m  > 1000 meters (but I used American spelling)

_In Glossary:_ 
Removed data-lt="Absolute positional accuracy"

PeterParslow commented:
Linda,
Thanks for trying to fix this.

I must admit I’ve now lost track of what is being compared to what, and what I might have successfully contributed already. So I’ll list everything that I can see (locally) / remember:

I also made changes related to British National Grid, around line 586 and 3033. This may have got in (somehow) via another pull request (660?). OSGB 1936 is the datum on which BNG (the projected reference system) is based, so at line 586 I suggest “and British National Grid (a national projected CRS, based on the OSGB 1936 datum) is” and 3033, just delete OSGB 1936

I reckon I changed a lot more than that, which I’ve attempted to highlight below (bold text).

From: Linda van den Brink [mailto:notifications@github.com]
Sent: 21 April 2017 10:06
To: w3c/sdw <sdw@noreply.github.com>
Cc: Peter Parslow <Peter.Parslow@os.uk>; Mention <mention@noreply.github.com>
Subject: Re: [w3c/sdw] Actioned changes in BP5 / section 12.2; also tweaked wording on Briti… (#673)


There have been other changes since this pull request so I compared the file from your zip with your previous version (I think).

The changes I found are listed below, could you check if this is correct? I have made these changes again in the current draft and will commit them soon.

In the intro on data quality (which has been merged with the intro on metadata):
This section is based on the Data Quality section from [[DWBP]] and adds a best practice specific for spatial data, which concentrates on the accuracy of the positions in the data - how close are they to the actual positions of the real world things?

Appended to the intro on metadata:

However, that specification makes no statement about the positional accuracy of the data,
so on its own, it is only a useful quality statement for users to whom positional accuracy is not that important.

[PP: ] that paragraph should appear after the GeoDCAT example which states conformance to an INSPIRE spec, which I moved up from a bit further down

In the Why section of former BP5:
Some spatial data applications, such as aircraft navigation, require highly accurate data. For others, such as human navigation, a horizontal accuracy of a few metres is good enough. For yet others, such as overlaying weather forecasts on a map, the map is only giving a general indication of place. If the positional accuracy is published together with the data, the user can determine whether it is appropriate to use for their application. Potentially, this makes existing data more re-usable.

[PP: ] I also edited the ‘intended outcome’ section, to be more outcome focussed (as suggested in the discussion):

“For many uses, the positional accuracy of the data is an important aspect of assessing
                  its fitness for purpose (quality).
                  As with other data quality statements, this can be a quantitative measure, a statement
                  of conformance to a standard or policy, or an assertion or report of fitness for a particular purpose.”

I then added a substantial set of paragraphs to ‘how’:

“              <p>For observed (measured) datasets, it is possible to make specific quantitative statements
                  about positional accuracy, based on knowledge of the equipment used to make the observations,
                  and any processing carried out.</p>
                <p>For <a>coverages</a>, the sampling distance is an effective way of indicating the amount of
                  detail in the dataset - this is one of the meanings of the term "resolution".
                  Alternatively, samples of the data could be independently checked against the real world,
                  and the results of that check reported.
                  Either way, this is usually a statement of <a>absolute positional accuracy</a>, but for some uses,
                  relative positional accuracy is more important.</p>
                <p>Positional accuracy measurements, whether observed or asserted based on process,
                  can be given using QualityMeasurement.</p>
                <p>For modelled datasets, for example in planning and construction, there is no 'real world' against
                  which to assess the positional accuracy - but relative positional accuracy can still be stated.</p>
                <p>For many uses, a statement of the amount of detail provided is sufficient to assess fitness for purpose;
                  examples include "level of detail" (building models), "navigational purpose" (marine navigation),
                  "equivalent scale" or "zoom level" (cartography).
                  Sometimes, this is expressed as if it were a statement of positional accuracy.</p>
                <p>These can be expressed in the same way as for non-spatial data, using for example the QualityAnnotation,
                  Standard, and QualityPolicy statements of [[VOCAB-DQV]].</p>”

Removed from former BP5 example list:
•  second quarter of the 9th century is approx. 825-850 but could be
823-852
[PP: ] also removed “describe spatial data quality with [[VOCAB-DQV]] and [[GeoDCAT-AP]].” as that’s not really an example, and is now described in ‘how’:
Added an example: “a hydrographic survey may conform to the 'special order' survey specification in IHO S-44”

                <p>The following example shows how DQV can express conformance to a specified positional accuracy</p>
            <pre class="example" id="ex-geodcat-ap-dataset-conformance-with-specification2" title="GeoDCAT-AP specification of a dataset conformance with IHO's S44">a:Dataset a dcat:Dataset ;
  dct:conformsTo &lt;<a href="https://www.iho.int/iho_pubs/standard/S-44_5E.pdf">https://www.iho.int/iho_pubs/standard/S-44_5E.pdf#Special</a>&gt; .

&lt;<a href="https://www.iho.int/iho_pubs/standard/S-44_5E.pdf">https://www.iho.int/iho_pubs/standard/S-44_5E.pdf</a>&gt; a dct:Standard , foaf:Document ;
  dct:title "IHO Standards for Hydrographic Surveys"@en ;
  dct:issued "2008-02-01"^^xsd:date.</pre>


In former BP5 example list:
1 000m > 1000 meters (but I used American spelling)
[PP: ] Changed the text of the coverage example from “can express the precision of” to “can express the amount of detail in” (given how we define & use the word ‘precision’)

Added to ‘how to test’:

<p>Check if the metadata contains at least one human and machine readable statement regarding positional accuracy </p>
<p>Check that the kind of statement is relevant to the kind of data, e.g. not an absolute positional accuracy measure for Atlantis</p>
<p>Checking that the accuracy statement is actually correct is beyond the scope of this best practice.</p>

In Glossary:
Removed data-lt="Absolute positional accuracy"

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/w3c/sdw/pull/673#issuecomment-296135581>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIGUoXBBfDHvWpBC0Tm21ZAxx7r28PKwks5ryHFzgaJpZM4M4oMk>.


This email is only intended for the person to whom it is addressed and may contain confidential information. If you have received this email in error, please notify the sender and delete this email which must not be copied, distributed or disclosed to any other person.

Unless stated otherwise, the contents of this email are personal to the writer and do not represent the official view of Ordnance Survey. Nor can any contract be formed on Ordnance Survey's behalf via email. We reserve the right to monitor emails and attachments without prior notice.

Thank you for your cooperation.

Ordnance Survey Limited (Company Registration number 09121572)
Registered Office: Explorer House
Adanac Drive
Southampton SO16 0AS
Tel: 03456 050505
http://www.os.uk

lvdbrink commented:
Thanks for listing it all! The only changes that got lost were the ones concerning OSGB 1936. I made those changes again in my version. Reviewed the rest you listed, everything is there. Must have been in your other PR which I merged succesfully. I think it's all good! 

You can check http://w3c.github.io/sdw/bp/ if you want to be sure it's all in there. 

Will close this PR now!

PeterParslow commented:
Linda, thanks for unpicking my mess.
